tdrs-frontend/src/components/SplashPage/SplashPage.jsx

```diff
@@ -13,11 +13,18 @@ import Button from '../Button'
 function SplashPage() {
   const authenticated = useSelector((state) => state.auth.authenticated)
   const authLoading = useSelector((state) => state.auth.loading)
+  const isInactive = useSelector((state) => state.auth.inactive)
+  const alertRef = useRef(null)
 
   const handleClick = (event) => {
     event.preventDefault()
     window.location.href = `${process.env.REACT_APP_BACKEND_URL}/login/oidc`
   }
+  useEffect(() => {
+    if (isInactive) {
+      setTimeout(() => alertRef?.current?.focus(), 2)
+    }
+  }, [alertRef, isInactive])
```

Comment:
Ensure that the Inactive Account banner is focused so that it can be read by screen readers when there is an `inactive` property in our `auth` state.